audio/audiotools/core/util.py

```diff
@@ -0,0 +1,933 @@
+import collections
+import csv
+import glob
+import math
+import numbers
+import os
+import random
+import typing
+from contextlib import contextmanager
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any
+from typing import Callable
+from typing import Dict
+from typing import Iterable
+from typing import List
+from typing import NamedTuple
+from typing import Optional
+from typing import Tuple
+from typing import Type
+from typing import Union
+
+import ffmpeg
+import librosa
+import numpy as np
+import paddle
+import soundfile
+from flatten_dict import flatten
+from flatten_dict import unflatten
+
+from .audio_signal import AudioSignal
+from paddlespeech.utils import satisfy_paddle_version
```

Comment:
If relative paths are used, they should be used uniformly

Reply:
Well, I think, if the content is from `audio_signal`, so use relative paths; and if it’s from `paddlespeech`, thus use absolute paths.